<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConnection</class>
 <widget class="QDialog" name="DialogConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="text">
      <string>&amp;Name</string>
     </property>
     <property name="buddy">
      <cstring>leName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lProtocol">
     <property name="text">
      <string>&amp;Protocol</string>
     </property>
     <property name="buddy">
      <cstring>cbProtocol</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbProtocol"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lHost">
     <property name="text">
      <string>&amp;Host</string>
     </property>
     <property name="buddy">
      <cstring>cbHost</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbHost"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lPort">
     <property name="text">
      <string>&amp;Port</string>
     </property>
     <property name="buddy">
      <cstring>sbPort</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbPort">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lVar">
     <property name="text">
      <string>&amp;Variable</string>
     </property>
     <property name="buddy">
      <cstring>cbVar</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbVar"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lArgs">
     <property name="text">
      <string>&amp;Args</string>
     </property>
     <property name="buddy">
      <cstring>leArgs</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="leArgs"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lComments">
     <property name="text">
      <string>&amp;Comments</string>
     </property>
     <property name="buddy">
      <cstring>leComments</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="leComments"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConnection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConnection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
